src/lib/tvgPaint.h

```diff
@@ -186,12 +182,12 @@ namespace tvg
             return ret;
         }
 
-        bool composite(Paint* target, CompMethod compMethod)
+        bool composite(Paint* target, CompositeMethod method)
         {
-            this->compTarget = target;
-            this->compMethod = compMethod;
-            if (this->compTarget) return true;
-            return false;
+            if (!target && method != CompositeMethod::None) return false;
```

Comment:
Is the condition correct?
Why do we need the method condition?
If we need to refer to method, I think it should be if (!target || method == CompositeMethod::None).